File: server/application/src/main/java/de/codecentric/iot/rapiro/blazeds/destinations/MovementPublishingActor.java
package de.codecentric.iot.rapiro.blazeds.destinations;

import de.codecentric.iot.rapiro.blazeds.actors.BlazeDsPublishingActor;
import org.springframework.context.annotation.Scope;
import org.springframework.stereotype.Component;

/**
 * Created by christoferdutz on 18.10.16.
 */
@Scope("prototype")
@Component("movementPublishingActor")
public class MovementPublishingActor extends BlazeDsPublishingActor {

    public MovementPublishingActor() {
        super("movementEvents");
    }

}

File: server/application/src/main/scala/de/codecentric/iot/rapiro/blazeds/destinations/MovementPublishingActor.scala
package de.codecentric.iot.rapiro.blazeds.destinations

import de.codecentric.iot.rapiro.blazeds.actors.BlazeDsPublishingActor
import org.springframework.context.annotation.Scope
import org.springframework.stereotype.Component

/**
  * Created by christoferdutz on 24.10.16.
  */
@Scope ("prototype")
@Component ("movementPublishingActor")
class MovementPublishingActor extends BlazeDsPublishingActor {

  override def destinationName: String = "movementEvents"

}
